imap_processing/codice/codice_l1a.py

```diff
@@ -165,8 +157,7 @@ def create_science_dataset(
                 "epoch": epoch,
                 "inst_az": inst_az,
                 "spin_sector": spin_sector,
-                "energy": energy_steps,
-                "energy_label": energy_label,
+                "esa_step": esa_step,
```

Comment:
If you remove those labels, you are gonna get error for data whose dimension have more than `epoch` as dimension when we need to update `cdflib`. To test it, some of us have been updating `cdflib` locally and then run the same test and it will raise those helpful errors. Can you do the same?

Eg.
```
dims = ["epoch", "inst_az", "spin_sector", "esa_step"]
```

Reply:
Ah yes, I forgot that I added `energy_label` in anticipation of the `cdflib` upgrade in the first place. But I think I found a new (better) solution, which is to define the `LABL_PTR_[1|2|3]`s in the attribute YAML file. It appears to work in both `cdflib` `1.3` and `1.2`.

Reply:
Good to know. I am surprised that you were able to reuse dimension cdf attrs for labels like this
```
    LABL_PTR_1: inst_az
    LABL_PTR_2: spin_sector
    LABL_PTR_3: esa_step
```
I thought it has to be metadata type.